tests/run-macros/i5941/macro_1.scala

```diff
@@ -102,8 +102,8 @@ object Iso {
     val cls = tpS.classSymbol.get
 
     val companion = tpS match {
-      case Type.SymRef(sym, prefix)   => Type.TermRef(prefix, sym.name)
-      case Type.TypeRef(name, prefix) => Type.TermRef(prefix, name)
+      case Type.TypeRef(sym, prefix) => Type.NamedTermRef(prefix, sym.name)
+      case Type.NamedTypeRef(name, prefix) => Type.NamedTermRef(prefix, name)
```

Comment:
As a macro author, usability can be improved here, given that `TypeRef` and `NamedTypeRef` both are backed by symbols, not structural type members.